src/common/components/RichTextEditor/index.js

```diff
@@ -104,7 +104,7 @@ const withInlines = editor => {
 
   editor.insertText = text => {
     if (text && isUrl(text)) {
-      wrapLinkPartial(editor)(transformURL(text), text);
+      wrapLinkPartial(editor)(transformURL(text).validTld(), text);
```

Comment:
Question: how is that validTld() function executing?, as far as I can see transformURL() returns a string

Reply:
I believe you're right. I thought it was validating the input, but looking at it now, I don't think the validTld() is necessary.